packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -377,6 +378,7 @@ export function toInstrumentedOdspTokenFetcher(
 	resolvedUrlParts: IOdspUrlParts,
 	tokenFetcher: TokenFetcher<OdspResourceTokenFetchOptions>,
 	throwOnNullToken: boolean,
+	returnPlainToken: boolean = false,
```

Comment:
nitpick: prefer to use explicit args without default value. Are you doing this to avoid breaking change? Is this member exported by the package?

Reply:
I don't think this is exported. Can make it required.

Reply:
Fixed